src/SourceBuild/content/build.proj

```diff
@@ -25,6 +27,21 @@
              StopOnFirstFailure="true" />
   </Target>
 
+  <Target Name="PublishVerticalManifest" AfterTargets="Build">
+    <PropertyGroup>      
+      <VerticalAssetManifestOutputPath>$(ArtifactsDir)VerticalManifest.xml</VerticalAssetManifestOutputPath>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <RepoAssetManifest Include="$(AssetManifestsIntermediateDir)\**\*.xml" />
+    </ItemGroup>
+
+    <PublishVerticalManifest 
+      AssetManifest="@(RepoAssetManifest)" 
+      VerticalAssetManifestOutputPath="$(VerticalAssetManifestOutputPath)"
```

Comment:
I would still call this task parameter MergedAssetManifestOutputPath instead. The Vertical name doesn't really mean anything from the task point of view. Vertical just refers to the nature of the VMR build.

Reply:
I renamed the parameter. Not sure if I should rename the whole task now tho